pkgs/development/python-modules/django-auth-ldap/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, ldap , django_2_2 
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "django-auth-ldap";
+  version = "1.7.0";
+
```

```suggestion
  disabled = isPy27;
```

This doesn't support that version.

---

fixed